internal/provider/resource_tfe_variable_set.go

```diff
@@ -147,6 +155,7 @@ func resourceTFEVariableSetUpdate(d *schema.ResourceData, meta interface{}) erro
 			Name:        tfe.String(d.Get("name").(string)),
 			Description: tfe.String(d.Get("description").(string)),
 			Global:      tfe.Bool(d.Get("global").(bool)),
+			Priority:    tfe.Bool(d.Get("priority").(bool)),
```

Comment:
currently, `priority` cannot be updated because the changes is not detected.

- [ ] TODO: update the HasChange chain in lines 153 `d.HasChange("name") || d.HasChange("description") || d.HasChange("global") || d.HasChange("priority")`